### ISSUE TITLE
MSSimulator and PTMs

### ISSUE THREAD

**Issue by lopippo** [OpenMS/OpenMS#2948]
Greetings,

I turned to MSSimulator to simulate mass spectra in the context of a mass spec teaching course. I was willing to model spectra for modified or unmodified proteins (acetylation on N-term and phosphorylations). Looking at the manual, I did not find a way to add this information. Is there a way to do it by somehow documenting the fasta file? Maybe, one option would be to define pseudo-amino acids, like B for phosphoserine, for example? Were would it be best to add these definitions?

Thank you for that useful program.
Sincerely,
Filippo

**Comment by cbielow** [comment]
Hi,

you can just add modifications to the FASTA file,

e.g.
```
>seq1 optional comment [# intensity=567.4 #]
VPTPNVSVVDLTC(Carbamidomethyl)R
```

This is not explicitly mentioned in the docs. A fix is coming up.
Note, the mods are fixed. If you require variable mods, you need to do the combinatorics 'manually'.

**Comment by cbielow** [comment]
Let me know if you run into any trouble.

**Comment by cbielow** [comment]
fixed in #2949 

closing for now. Re-open if something does not work

**Comment by lopippo** [comment]
Fine, thank you. I'll test that next week.
